### PR TITLE
Add autogenerated README to the chart

### PR DESCRIPTION
**Pull request by kang-makes**

### CODE REVIEW

File: charts/nri-metadata-injection/README.md [hunk 1]
```diff
@@ -1,64 +1,75 @@
...
-| `dnsConfig`                      | Set DNS settings                                                                                                                                                                                                                                                                                                      | `{}`                                |
+# Helm installation
 
-## Example
+You can install this chart using [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) located in the
+[helm-charts repository](https://github.com/newrelic/helm-charts) or directly from this repository by adding this Helm repository:
 
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
+```shell
+helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
```
Comment (paologallinaharbur) [inline charts/nri-metadata-injection/README.md:15]: copy-paste issue

File: charts/nri-metadata-injection/README.md [hunk 2]
```diff
@@ -1,64 +1,75 @@
...
+helm upgrade --install nri-kube-events/nri-kube-events -f your-custom-values.yaml
```
Comment (paologallinaharbur) [inline charts/nri-metadata-injection/README.md:16]: here also

File: charts/nri-metadata-injection/values.yaml [hunk 3]
```diff
@@ -1,81 +1,100 @@
...
+# -- Override the full name of the release
+fullnameOverride: ""
 
-# Enable the metadata decoration only for pods living in namespaces labeled
-# with 'newrelic-metadata-injection=enabled'.
-injectOnlyLabeledNamespaces: false
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`
+cluster: ""
+# -- This set this license key to use. Can be configured also with `global.licenseKey`
+licenseKey: ""
```
Comment (paologallinaharbur) [inline charts/nri-metadata-injection/values.yaml:9]: license key is not used in this chart, also customSecretName and customSecretLicenseKey should not be set

Comment (paologallinaharbur) [reply charts/nri-metadata-injection/values.yaml:9]: Unless something changed

Comment (kang-makes) [reply charts/nri-metadata-injection/values.yaml]: You are right. Solved in [#4a082e6](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/4a082e6389a9968fb77a01508b68f2da71551ca0)

File: charts/nri-metadata-injection/README.md (hunk 2 above)
Comment (kang-makes) [reply charts/nri-metadata-injection/README.md]: Solved in [#4a082e6](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/4a082e6389a9968fb77a01508b68f2da71551ca0)

File: charts/nri-metadata-injection/README.md (hunk 1 above)
Comment (kang-makes) [reply charts/nri-metadata-injection/README.md]: Solved in [#4a082e6](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/4a082e6389a9968fb77a01508b68f2da71551ca0)

File: charts/nri-metadata-injection/values.yaml [hunk 4]
```diff
@@ -3,15 +3,6 @@ nameOverride: ""
 # -- Override the full name of the release
 fullnameOverride: ""
 
-# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`
-cluster: ""
```
Comment (paologallinaharbur) [inline charts/nri-metadata-injection/values.yaml]: this is needed I believe because that info is injected as env var

Comment (kang-makes) [reply charts/nri-metadata-injection/values.yaml]: Solved in [#e4fba9f](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/e4fba9f678497857c8c3ba587ae9be175ed5398d)

File: charts/nri-metadata-injection/values.yaml [hunk 5]
```diff
@@ -1,81 +1,94 @@
...
+# -- Webhook timeout
 # Ref: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts
-timeoutSeconds: 30
-
-# Use cert manager for webhook certs
-certManager:
-  enabled: false
-
-tolerations: []
+timeoutSeconds: 28
```
Comment (paologallinaharbur) [inline charts/nri-metadata-injection/values.yaml:94]: That was due to an issue, right? Can we add a link into the code?

**Review by paologallinaharbur — APPROVED** [review]

File: charts/nri-metadata-injection/values.yaml (hunk 5 above)
Comment (kang-makes) [reply charts/nri-metadata-injection/values.yaml:94]: It is already. The `Ref` link goes to a doc that says that the value should between 0 and 30 (not included)

**Pull request merged**